pocld/request.hh

```diff
@@ -56,48 +56,48 @@ private:
 class Request {
 public:
   /// Size, in bytes, of the main request body (up to sizeof RequestMsg_t)
-  uint32_t BodySize;
+  uint32_t BodySize = 0;
```

Comment:
It might be best to {} initialize in the instantiator side than rely on these member variable initializers.  https://en.cppreference.com/w/cpp/language/zero_initialization